src/App.vue

```diff
@@ -5,7 +5,7 @@
     <div class="wrap-outer-content layout-band">
       <div class="wrap-content">
         <div class="layout-band">
-          <SearchForm msg="This is the search form." />
+          <SearchForm @searched="doSearch" />
```

Comment:
I notice you use `@` here and `v-on` elsewhere. I think we should have a team conversation (maybe at today's meeting?) about how we use Vue's syntactic sugar and whether it matters if it's consistent throughout the codebase. My sense is that we should use it either everywhere or nowhere, but that's not a strongly held opinion.

Reply:
Trying to summarize what we discussed in our group meeting - for now we're choosing not to have a preference for using syntactic sugar because we don't have enough collective experience to prefer using it or not. If one of us gets to the point of having a strong preference, we can have that discussion then.

Thanks for mentioning this as an area for us to be cognizant of, though.